play/src/front/Chat/Components/SelectMatrixUser.svelte

```diff
@@ -0,0 +1,99 @@
+<script lang="ts">
+    import { onMount, createEventDispatcher } from "svelte";
+    import * as Sentry from "@sentry/svelte";
+    import Select from "svelte-select";
+    import LL from "../../../i18n/i18n-svelte";
+    import { gameManager } from "../../Phaser/Game/GameManager";
+    import { searchChatMembersRule } from "./Room/searchChatMembersRule";
+
+    export let value: { value: string; label: string; verified?: boolean }[] = [];
+    export let placeholder = "";
+    export let filterText = "";
+    let items: { value: string; label: string | number | undefined; created?: boolean; verified?: boolean }[] = [];
+    const chat = gameManager.chatConnection;
+
+    const dispatch = createEventDispatcher();
+    const { searchWorldMembers } = searchChatMembersRule();
+
+    function handleFilter(e: CustomEvent) {
+        if (value?.find((i) => i.label === filterText)) return;
+        if (e.detail.length === 0 && filterText.length > 0) {
+            const prev = items.filter((i) => !i.created);
+            items = [...prev, { value: filterText, label: filterText, created: true }];
+        }
+    }
+
+    async function handleChange() {
+        const verificationResults = await Promise.all(
+            value.map(async (item) => {
+                if (item.verified !== undefined) {
+                    return { item, isValid: item.verified };
+                }
+                try {
+                    const isValid = await chat.isUserExist(item.value);
+                    return { item, isValid };
+                } catch (error) {
+                    console.error(error);
+                    return { item, isValid: false };
+                }
+            })
+        );
+
+        const validItems = verificationResults
+            .filter(({ isValid }) => isValid)
+            .map(({ item }) => ({ ...item, verified: true }));
+
+        const hasInvalidItems = verificationResults.some(({ isValid }) => !isValid);
+        if (hasInvalidItems) {
+            dispatch("error", { error: "User not found" });
+        }
+
+        return validItems;
+    }
+
+    async function loadItems() {
+        try {
+            return await searchWorldMembers(filterText);
+        } catch (error) {
+            console.error(error);
```

Comment:
If there is an error when we try to search world member, could we display this error?